.circleci/config.yml

```diff
@@ -1,4 +1,18 @@
----
+version: 2.1
+
+executors:
+  test-bundle:
+    docker:
+      - image: cimg/base:2021.04
+        environment:
+          FOO: $foo
+
+      - image: alpine
+        environment:
+          FOO: $foo
+        # Need foo env var here, but getting literal as $foo
+        command: ["sh", "-c", "env"]
```

Comment:
No env passed inside the sub container

Reply:
@buchs ^^

Reply:
What this suggests to me is that the environment variable "foo" was not set in the Circle CI environment variables for the project.

Reply:
![2021-05-13_21-53](https://user-images.githubusercontent.com/797361/118214791-0d577080-b436-11eb-8ae2-94f1245a5460.png)

Reply:
It was, I believe, but good point let me doublecheck that.

Reply:
Yes it is set up. I can see env variable in the host but not in sub container.
<img width="1276" alt="bundle__18__-_dmi3y_circle-ci-test" src="https://user-images.githubusercontent.com/1277279/118534725-678a5700-b6fe-11eb-9898-fa6181385ff1.png">

Reply:
Oh, and what I figured out it does not work the host as well, at least for build job...
<img width="1278" alt="Cursor_and_build__3__-_buchs_circle-ci-test" src="https://user-images.githubusercontent.com/1277279/118535989-03689280-b700-11eb-8cd2-6135a119e5f5.png">